src/components/ContentTemplate.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<NcAppContent :show-details="showDetails"
+		@update:showDetails="unselectItem()">
+		<template #list>
+			<NcAppContentList>
+				<div class="header">
+					<slot name="header" />
+				</div>
+
+				<FeedItemDisplayList :items="items"
+					:fetch-key="fetchKey"
+					:config="config"
+					@load-more="emit('load-more')" />
+			</NcAppContentList>
+		</template>
+
+		<NcAppContentDetails class="content-details-container">
+			<FeedItemDisplay v-if="selectedFeedItem" :item="selectedFeedItem" />
+		</NcAppContentDetails>
+	</NcAppContent>
+</template>
+
+<script setup lang="ts">
+
+import { type PropType, computed, ref, watch } from 'vue'
+
+import itemStore from '../store/item'
+
+import NcAppContent from '@nextcloud/vue/dist/Components/NcAppContent'
+import NcAppContentList from '@nextcloud/vue/dist/Components/NcAppContentList'
+import NcAppContentDetails from '@nextcloud/vue/dist/Components/NcAppContentDetails'
+
+import { FeedItem } from '../types/FeedItem'
+
+import FeedItemDisplayList from './feed-display/FeedItemDisplayList.vue'
+import FeedItemDisplay from './feed-display/FeedItemDisplay.vue'
+
```

Comment:
could we maybe add a comment that this file use the composition API and provide the link: https://vuejs.org/guide/extras/composition-api-faq.html

Reply:
Sure.